### ISSUE TITLE
Upgrade to bootstrap 4 and better meeting WCAG3 guidelines

### ISSUE THREAD

**Issue by Pierre-Sassoulas** [Pierre-Sassoulas/django-survey#109]
> I would propose going a step further and creating a more accessible stylesheet than the current implementation. The grey backdrop doesn't meet the contrast ratio required for WCAG 3 accessibility guidelines - I know this is a minor suggestion, possibly due to use case (patient research)/HIPAA compliance background, but it can't hurt?

https://github.com/Pierre-Sassoulas/django-survey/issues/108#issuecomment-725352604

**Comment by d-horner** [comment]
@Pierre-Sassoulas would you be open to including third-party css/js libraries from a CDN like cdnjs's one vs serving it locally? Typically increases performance and decreases bandwidth vs serving the static files from within the project - with browsers caching the latest libraries for such popular frameworks/libs. Obviously, users of the project can override the base templates regardless.

**Comment by Pierre-Sassoulas** [comment]
I think it could become a setting option being opt-in, instead of the default, because the performance gain is not guaranteed and it could permit to inject malicious code. If creating the option is too much work, let's just make the override of the template easier.
